dockerfiles/hpcc/templates/roxie.yaml

```diff
@@ -1,4 +1,5 @@
 {{ range $roxie := $.Values.roxie -}}
+{{- if $roxie.enabled -}}
```

Comment:
Note: added these since the old trick of renaming roxie section to xroxie to speed things up when testing other parts of system will now be rejected